Core/include/Acts/Utilities/BinAdjustmentVolume.hpp

```diff
@@ -76,6 +80,62 @@
   return uBinUtil;
 }
 
+/// @brief adjust the BinUtility bu to the dimensions of cylinder volume bounds
+///
+/// @param bu BinUtility at source
+/// @param cBounds the Cylinder volume bounds to adjust to
```

Comment:
Shouldn't that mention cutout cylinder volumes explicitly?

Reply:
Done